lib/iris/fileformats/grib/__init__.py

```diff
@@ -47,80 +48,20 @@
 from iris.fileformats.grib._message import _GribMessage
 import iris.fileformats.grib.load_rules
 import iris.unit
+from iris.fileformats.grib._gribwrapper import (GribWrapper, grib_generator,
+                                                GribDataProxy)
 
-
-__all__ = ['grib_generator', 'load_cubes',
-           'reset_load_rules', 'save_grib2', 'GribWrapper',
-           'hindcast_workaround']
+__all__ = ['load_cubes', 'reset_load_rules', 'save_grib2']
```

Comment:
You've removed `hindcast_workaround` - was that intentional? Strictly speaking it's a backwards-incompatible change - it could make someone's code break.